src/main/java/org/apache/ibatis/jdbc/AbstractSQL.java

```diff
@@ -310,6 +314,11 @@ public T OFFSET(long value) {
     return getSelf();
   }
 
+  public T ADD_ROW() {
```

Comment:
Please add `@Since 3.5.2`

Reply:
- [ ] Please check a position of JavaDoc for `ADD_ROW` (Probably, there is mistake for your merging).

- [ ] JavaDoc's item order is not standard. Standard order is "description -> `@since`" in this case. But currently "`@since` -> description".

- [ ] Please add `"."`(period) end with sentence of description.

- [ ] Please change to `@since` from `@Since` (`S` change to lower case)